torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3228,6 +3274,10 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypesIfCUDA=all_types_and(torch.float16, torch.bfloat16, torch.bool),
            supports_out=False,
            sample_inputs_func=sample_inputs_max_min_reduction_no_dim,),
+    OpInfo('sum',
+           dtypes=all_types_and_complex_and(torch.float16, torch.bfloat16, torch.bool),
```

Comment:
should we also compare this against the numpy function: https://numpy.org/doc/stable/reference/generated/numpy.sum.html ?

Reply:
The base OpInfo class doesn't support references, unfortunately. I'm sure ReductionInfos will, though ;)